javascript/code-challenges/trees/binary-tree/binary-tree.js

```diff
@@ -69,6 +69,45 @@ class BinaryTree{
     return results
   }
 
+  fizzBuzz() {
+    if(this.root === null) { throw new Error('No Nodes Present in Binary Tree')}
+
+    // Copy Tree
+    let treeCopy = this
```

Comment:
This doesn't correctly copy tree. The changes to the copy are reflected in this if I remember correctly.

Reply:
Fixed in aeaf0cd49686ee62a643fa5855d75d1c8912c8e5!